fusor-ember-cli/app/controllers/deployment-new/satellite/configure-environment.js

```diff
@@ -32,12 +32,6 @@ export default Ember.Controller.extend(ConfigureEnvironmentMixin, NeedsDeploymen
       return this.get('deploymentNewController.model').set('lifecycle_environment', environment);
     },
 
-    newEnvironment() {
-      this.set('newEnvName', '');
-      this.set('description', '');
-      this.set('openNewEnvironmentModal', true);
```

Comment:
this should have been `openModal` rather than `openNewEnvironmentModal`, but I decided to put the action in the mixin to DRY things up.